### ISSUE TITLE
Tech Working Group Updates

### ISSUE THREAD

**Issue by smit1678** [hotosm/tech#117]
At FOSS4G/HOT Summit we heard a number of things from staff, members, and the community. One common theme that came out was the need to propose some changes to the Tech Working Group to meet demand of development pace and look to get more people involved in specific tools and not just admin processes.

Over the past year, the Tech Working Group meeting has been meeting bi-weekly and exploring some slightly different formats. While we’ve experimented with different formats (type of meetings, meetings in Asia and EU/Americas timezones), it’s mostly been an admin meeting with lower attendance. We’ve continued to receive feedback that the meeting is hard to get involved (it’s texted based) and that it would be nice to have more presentation style updates to see features demo’d or updates communicated.

## Updates to Tech Working Group

So we’re proposing a couple changes to how the Tech Working Group meets.

1. Look to go monthly voice meeting. Focus on sharing tools, data work, or other technical work from across projects, countries, people. We will need screen-sharing functionality and so we’ll look to use Google Hangouts Meet or Zoom for now and can consider some other open source options that can handle larger groups.
2. Add topical or tool based sub-working groups. Start with the following (but could be revisited next year):
  - Tasking Manager and ML
  - Data analysis and quality
  - Field collection tools

### Monthly Meeting

We’ll have our first month Tech Working Group meeting on Sept 25 at 15:00 UTC.
We’ll demo MapCampaigner and give a short training on how to use it and some future development ideas.

### Sub working group meetings

We’ll start our first Tasking Manager subworking Group meeting next week. It is looking like it will be Sept 21 at 15:00 UTC. 

Other thoughts or feedback on these changes?

**Comment by jmarlena** [comment]
Are meetings still happening?

**Comment by mataharimhairi** [comment]
@jmarlena yes they are! the next one is scheduled for 5th November at 16.00 UTC. Here is the Google Hangouts link if you are planning on participating:

https://meet.google.com/rmn-tygd-aab
